modules/n1ql/pages/n1ql-language-reference/reservedwords.adoc

```diff
@@ -11,7 +11,7 @@ Some of the keywords are not currently implemented but are reserved for future u
 == Using Reserved Words as Identifiers
 
 N1QL allows escaped identifiers to overlap with keywords.
-To use a reserved word as an identifier, you must escape it by enclosing the reserved word inside backticks (``).
+To use a reserved word as an identifier, you must escape it by enclosing the reserved word inside backticks (```).
```

Comment:
That works, but looking at the original text I guess there are supposed to be two backticks?
I always use a predefined character attribute when I want an actual backtick to appear: {backtick}
So you could try `{backtick}{backtick}`